### PR TITLE
new koji_util module for common koji handling

### PR DESCRIPTION
**Pull request by twaugh**

### CODE REVIEW

**Comment by lcarva** [comment]
LGTM

**Comment by mmilata** [comment]
LGTM

File: atomic_reactor/plugins/exit_koji_promote.py [hunk 1]
```diff
@@ -574,26 +575,11 @@ def login(self):
         :return: koji.ClientSession instance, logged in
         """
         session = koji.ClientSession(self.kojihub)
```
Comment (vrutkovs) [inline atomic_reactor/plugins/exit_koji_promote.py]: Nitpick: probably ```koji_login``` should prepare and return ```session``` object?

Comment (lcarva) [inline atomic_reactor/plugins/exit_koji_promote.py]: Good point. Maybe a `create_session(bool:login)` method that creates the session and conditionally performs login?

**Comment by lcarva** [comment]
Oops. Closed the wrong PR.

**Comment by twaugh** [comment]
LGTM, merging

**Pull request merged**